src/lib/doctrine-dbal-bulk/src/Flow/Doctrine/Bulk/Dialect/MySQLDialect.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\Doctrine\Bulk\Dialect;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Flow\Doctrine\Bulk\BulkData;
+use Flow\Doctrine\Bulk\Columns;
+use Flow\Doctrine\Bulk\Exception\RuntimeException;
+use Flow\Doctrine\Bulk\TableDefinition;
+
+final class MySQLDialect implements Dialect
+{
+    private AbstractPlatform $platform;
+
+    public function __construct(AbstractPlatform $platform)
+    {
+        $this->platform = $platform;
+    }
+
+    /**
+     * @param TableDefinition $table
+     * @param BulkData $bulkData
+     * @param array{
+     *  skip_conflicts?: boolean,
+     *  constraint?: string,
```

Comment:
this does not seems to be used anywhere?

Reply:
Fixed.